vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -63,13 +85,8 @@ def validate_multi(cls, multi, values):
     @_log_call
     def build(self):
         full_options, default_value = get_options_and_default(options=self.options, multi=self.multi)
-        # 30 characters is roughly the number of "A" characters you can fit comfortably on a line in the dropdown.
-        # "A" is representative of a slightly wider than average character:
-        # https://stackoverflow.com/questions/3949422/which-letter-of-the-english-alphabet-takes-up-most-pixels
-        # We look at the longest option to find number_of_lines it requires. Option height is the same for all options
-        # and needs 24px for each line + 8px padding.
-        number_of_lines = math.ceil(max(len(str(option)) for option in full_options) / 30)
-        option_height = 8 + 24 * number_of_lines
+        list_of_labels = _get_list_of_labels(full_options)
+        option_height = _calculate_option_height(list_of_labels)
```

Comment:
Perhaps calling `_get_list_of_labels` directly from `_calculate_option_height` is a better fit as we keep the `Dropdown.build` method simpler and more readable.